tsconfig.json

```diff
@@ -0,0 +1,67 @@
+{
+  "include": ["src/js/**.*s"],
```

Comment:
VSCode gives us a great idea as to what these options do – I think we can get rid of the comments.